components/03-sections/02-navigation/04-side-navigation-news/side-navigation-news.hbs

```diff
@@ -0,0 +1,91 @@
+<!-- Side Navigation -->
+<div class="sidebar-news">
+    {{!-- Related Stories Links --}}
+    <div class="sidebar-news-container">  
+        <div class="sidebar-news-title">Related News</div>
```

Comment:
It seems that the semantically correct way to handle this is to use an <aside></aside> html element to house the entire sidebar (instead of a div tag). Other than that, we can style the text as we wish, but I think I'd try to use h tags if possible and customize them to size them correctly.